.github/workflows/check-outdated-terms.yaml

```diff
@@ -61,7 +52,7 @@ jobs:
         echo "OUTPUT_DIR=${OUTPUT_DIR}" >> $GITHUB_ENV
     
     - name: Checkout 
-      uses: actions/checkout@v3
+      uses: actions/checkout@v3.5.3
```

Comment:
I personally prefer not to specify minor and patch versions for GitHub Action. Specifying those will incur relatively frequent PRs by bot to bump minor and patch versions every time the newer version comes.
In contrast, not specifying minor and patch versions and specifying major version only intentionally will
- make GitHub workflow pull the latest version (e.g., 3.5.3) of specified major version, and
- cause fewer PRs by bot, only when the new major version has been released.

You can see the PRs by bot to automatically bump the version of GitHub Actions at:
https://github.com/cncf/glossary/pulls?q=is%3Apr+is%3Aclosed+bump

What's your opinion? 😊

Reply:
My preference is `Version pinning`. It's a best practice [for dependency management](https://cloud.google.com/blog/topics/developers-practitioners/best-practices-dependency-management) that facilitates the troubleshooting. Several tools recommend this approach, like [Hadolint](https://github.com/hadolint/hadolint/wiki/DL3008) with Dockerfiles. I agree, that can be a like bit noisy, and sometimes makes a [bot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot) the top contributor of the project.

Reply:
@electrocucaracha We discussed about this in the monthly meeting yesterday, and reached a conclusion:
yes it's important to pin the specific version for dependency management and so on, but we maintainers prefer not to specify minor and patch versions in this particular case, because of some reasons, where part of them might be:
- the workflows configured in this repo are somewhat auxiliary ones, so we do not need very strict dependency management.
- the GitHub Actions that we are using seem to not be likely to cause (crucial) trouble.
- as you said it might be noisy.. 😅

And these reasons might apply to our case only, so it will be encouraged to pin the version for other general cases.

PTAL!